internal/bminventory/inventory_test.go

```diff
@@ -1516,6 +1516,33 @@ var _ = Describe("cluster", func() {
 				})
 			})
 
+			Context("Day2 api vip dnsname/ip", func() {
```

Comment:
how about testing `RegisterAddHostsCluster`?

Reply:
there is already a test that does it. the function was just re-formatted